action.yml

```
@@ -38,23 +38,55 @@ inputs:
     required: false
 
   runner-version:
-    description: "The version of the runner to use. Use 'latest' to automatically fetch the latest release version from GitHub, or specify a version like '3.5.0' or 'v3.5.0'."
+    description: "The version of the runner to use. Use 'latest' to automatically fetch the latest release version from GitHub, or specify a version like '3.5.0' or 'v3.5.0'. For debugging: 'branch:branch-name' or 'ref:commit-sha'."
```

This shouldn't be mentionned here since it's for internal use
```suggestion
    description: "The version of the runner to use. Use 'latest' to automatically fetch the latest release version from GitHub, or specify a version like '3.5.0' or 'v3.5.0'.
```